statics/js/components/graph-layout.js

```diff
@@ -402,6 +409,7 @@ TopologyGraphLayout.prototype = {
   },
 
   onNodeAdded: function(node) {
+    if (this.hidden(node)) return;
```

Comment:
specific field are added in  the _onNodeAdded function especially _metadata which is used at some places. Not adding it will trigger exceptions. I can either move this return in the _onNodeAdded or check where _metadata is used.